sdk/iot/core/tests/cmocka/az_precondition_testing.h

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@ewertons this file can't be placed in the `iot` folder as it doesn't follow the naming guidelines.
Is it common? Should it go to `core` instead?